### ISSUE TITLE
processing 3.x support

### ISSUE THREAD

**Issue by MauroMombelli** [carykh/evolutionSteer#11]
The size() function is required to be in the function "settings()" as  https://processing.org/reference/settings_.html

I have no idea how to make dual support for processing 2 and 3, but as processing stop the execution indicating the size() function call, you may add a little comment to explain the issue.

Your future yourself will thank you when you'll update processing

**Comment by Lomion-tm** [comment]
just put `void settings() {
  size((int)(windowWidth*windowSizeMultiplier), (int)(windowHeight*windowSizeMultiplier),P3D);
}
` outside of the function^^ and del the line causing the error...

**Comment by MauroMombelli** [comment]
yes, i did, the point is as is the only modification needed, would be nice to fix in the same way for all the version..

**Comment by hanss314** [comment]
Processing 2 code and Processing 3 code is incompatible and it's a pain to develop for both, just install 2 versions of processing.

**Comment by MauroMombelli** [comment]
lol, why keep around the OLD version? what is the advantage? at least the new is going to have support

**Comment by calvinjuarez** [comment]
We need a new branch for v3 is all.
